test_check_limits.py

```diff
@@ -0,0 +1,17 @@
+from check_limits import Check_Limits
+
+test_obj = Check_Limits()
+
+if __name__ == '__main__':
+    assert(test_obj.battery_is_ok([{25: True}, {70: True}, {0.76: True}], language='German') is True)
+    assert(test_obj.battery_is_ok([{50: True}, {85: True}, {0: True}]) is False)
```

Comment:
How do we verify that a particular language string is printed correctly? is there any test to verify that?
if not then there is scope for improvement here...

Reply:
Sure Sir, will try improvising the Test Cases